javascript/javascript3/week2/game.js

```diff
@@ -0,0 +1,42 @@
+const apiEndpoint = 'https://api.exchangerate-api.com/v4/latest/EUR';
+let exchangeRates, currencies;
+
+function convert() {
+  const amount = document.getElementById('amount').value;
+  const from = document.getElementById('from').value;
+  const to = document.getElementById('to').value;
+  const result = amount * exchangeRates[to] / exchangeRates[from];
+  document.getElementById('result').innerHTML = `${amount} ${from} = ${result} ${to}`;
+}
+
+function populateCurrencies() {
+  currencies = Object.keys(exchangeRates);
+  const fromSelect = document.getElementById('from');
+  const toSelect = document.getElementById('to');
+  currencies.forEach(currency => {
+    const option = document.createElement('option');
+    option.value = currency;
+    option.text = currency;
+    if (currency === 'EUR') {
+      option.selected = true;
+    }
+    fromSelect.appendChild(option);
+  });
+  currencies.forEach(currency => {
+    const option = document.createElement('option');
+    option.value = currency;
+    option.text = currency;
+    if (currency === 'DKK') {
+      option.selected = true;
+    }
+    toSelect.appendChild(option);
+  });
+}
+
+fetch(apiEndpoint)
+  .then(response => response.json())
+  .then(data => {
+    exchangeRates = data.rates;
```

Comment:
I would suggest defining `exchangeRates` as a local variable here, and then passing it to `populateCurrencies` as a parameter. 

In general it is better to avoid using global variables, since they may lead to unexpected behavior. Instead, try to use function parameters and return values as much as possible.